frontend/src/pages/ChangePaymentPage.tsx

```diff
@@ -0,0 +1,81 @@
+import {Navigate, useNavigate, useParams} from "react-router-dom";
+import React, {ChangeEvent, FormEvent, useContext, useMemo, useState} from "react";
+import {DatePicker, LocalizationProvider} from "@mui/lab";
+import AdapterDateFns from "@mui/lab/AdapterDateFns";
+import {TextField} from "@mui/material";
+import {IPaymentController} from "../models/ControllerTypes";
+import PaymentController from "../controllers/PaymentController";
+import {AuthContext} from "../context/AuthProvider";
+import {IPayment} from "../models/IPayment";
+import './ChangePaymentPage.scss'
+
+export default function ChangePaymentPage() {
+    const params = useParams()
+    const categoryID = params.categoryID
+    const paymentID = params.paymentID
+    const [description, setDescription] = useState<string>("")
+    const [amount, setAmount] = useState<number>(0)
+    const [date, setDate] = useState<Date | null>(new Date(Date.now()))
```

Comment:
Wenn es nicht die möglichkeit geben soll kein Datum auszuwählen würde ich den Typ auch entsprechend setzten.

```suggestion
    const [date, setDate] = useState<Date>(new Date(Date.now()))
```